templates/gallery/partials/header.snip.html

```diff
@@ -0,0 +1,49 @@
+{{!
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+{{#header}}
+  <header
+    class="
+      gallery-header
+      {{#invert}}gallery-base-invert{{/invert}}
+      {{#mobile-header}}gallery-mobile-header{{/mobile-header}}
+    "
+  >
+    <button
+      class="ampstart-btn gallery-header-menu-button"
+      role="button"
+      tabindex="-1"
+      on="tap:sidebar.open"
+    >
+      <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
```

Comment:
may be move to img file and import from there - Like we did here https://github.com/ampproject/ampstart/blob/master/components/social-follow/social-follow.snip.html#L33

Reply:
Similar to the above comment, I think will change because of our Sidebar PR changes, I'll update here once I figure out the plan.

Reply:
I guess if comments are addressed we can merge this and you can always delete this on future PRs. This makes sense if there are other useful parts in this PR